client/galaxy/scripts/mvc/ui/ui-select-genomespace.js

```diff
@@ -76,17 +66,14 @@ var View = Backbone.View.extend({
 
     // get value
     _getValue: function() {
-        return `${this.filename_textbox.value()}^${this.token_textbox.value()}`;
+        return this.filename_textbox.value();
     },
 
     // set value
     _setValue: function(new_value) {
         if (new_value) {
-            var values = new_value.split("^");
-            this.filename_textbox.value(values[0]);
-            this.token_textbox.value(values[1]);
+            this.filename_textbox.value(new_value);
         }
-        this.options.onchange && this.options.onchange(new_value);
```

Comment:
Not sure what it does, but this line was recently added in 8c941ae35a37f8dd087222c6827c122ddfdcc966, is it OK to remove it?

Reply:
that should actually stay, good catch

it triggers the onchange event if it exists